src/library/scala/util/Either.scala

```diff
@@ -457,6 +457,35 @@ object Either {
   def cond[A, B](test: Boolean, right: => B, left: => A): Either[A, B] =
     if (test) Right(right) else Left(left)
 
+  /**
+   * Creates `FromLeftPartiallyApplied[B]` that given an argument of type `A`
```

Comment:
Since this name appears in the Scaladoc, perhaps a shorter name could be found?

Reply:
Do you have any good ideas? I think the name is long, but very descriptive